torchani/data/new.py

```diff
@@ -27,8 +27,8 @@ class CachedDataset(torch.utils.data.Dataset):
         file_path (str): Path to one hdf5 files.
         batch_size (int): batch size.
         chunk_threshold (int): threshould to split batch into chunks.
-        species_order (str): a string which specify how species are transfomed to int.
-            for example: 'HCNO' means {'H': 0, 'C': 1, 'N': 2, 'O': 3}
+        species_order (list): a list which specify how species are transfomed to int.
```

Comment:
not "list", please use "sequence", anything that you could iterate is fine, such as tuple, list, etc. The word "sequence" is a general word for all those kind of things

Reply:
tuple is fine, but it should be fixed. some people may not know what is tuple, what is list.

Reply:
that's what I mean, it should be fixed, people don't familiar python don't know what to use, in the docs, it should be specified for them

Reply:
OK, not good, but acceptable